### PR TITLE
Comment for course link

### PR DESCRIPTION
**Pull request by son-of-mountain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ This repo is a series of workshops from the html and css course by openclassroom
 `git clone https://github.com/son-of-mountain/RobbieLens-portfolio.git`
 # Info about the course provider
 The french online school is OpenClassrooms
-
+# Link to the course
```
Comment (son-of-mountain) [inline README.md:14]: Nice work but you forgot the actual link smh !

**Review by son-of-mountain — COMMENTED** [review]
Nice work but you forgot the actual link smh !

**Review by mouaadsgalaxy — COMMENTED** [review]
Nice work but you forgot the actual link smh !

**Comment by son-of-mountain** [comment]
Oh my bad!

File: README.md [hunk 2]
```diff
@@ -11,4 +11,5 @@ This repo is a series of workshops from the html and css course by openclassroom
...
+please check `openclassrooms.com`
```
Comment (mouaadsgalaxy) [inline README.md:15]: Good job you added the link

**Pull request merged**